acceleration/TensorRT_inference_acceleration.ipynb

```diff
@@ -19,17 +19,15 @@
    ]
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    spatial_shape = (1, 3, 736, 480)
</pre><p>spatial_shape is not used in this cell. Maybe we can move it to the next code cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>This part is more like a parameter definition of the bundle. So I put these parameters here. Users may want to use other bundles with different <code>net_id</code>, <code>spatial_shape</code> , config file name and so on. I think it would be more covenient to put these parameters together. Then users only need to update parameters in this cell to run the tutorial. What do you think about it?</p><p> </p><p>Thanks,</p><p>Bin</p>

Reply:
Putting it here makes sense to me. Thanks!